### ISSUE TITLE
These should be a button, not inside a <div>

### ISSUE THREAD

**Issue by ciaoaaron** [vmzeller/ZenSched#3]
https://github.com/vmzeller/ZenSched/blob/8fac5ddace613f87da3b16b5fcd62d732fdb8e72/index.html#L50-L65

For a screen reader user, screen reader should announce these as a button. Using `<div>` would just let the screen reader to read them as text and not tell the user that it's a button.
